data/event_config.yml

```diff
@@ -1,5 +1,7 @@
 twitter_handle:
    default: 
       name: "JakartaOneConf"
-   japan2020: 
+   japan: 
       name: "JakartaOneJPN"
+   russia: 
+      name: "JakartaOneRUS"
```

Comment:
I'm changing this because the Japanese and Russian twitter were not correctly rendered.